troubleshooting/troubleshooting-additional-digit-appended-to-apk.md

```diff
@@ -16,15 +16,15 @@ pitsid:
 <table>
   <tr>
     <td>Service</td>
-    <td>Code, Distribution</td>	
+    <td>Code (AppBuilder), Distribution</td>	
```

Comment:
AppManager doesn't use the version code (only the version string), so I think you may remove it from the Service list here.